### PR TITLE
Collect metrics for all subgraphs and generate input variation tables for each model

### PR DESCRIPTION
**Pull request by kevinwuTT**
- [x] Links to info directories for some models like MNIST are broken
- [x] Simplify calling the model again with different types of inputs
- [x] Add some comments to new functions in collect_metrics.py

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,176 +45,6 @@ The table below summarizes the results of running various ML models through our
...
-| aten ops                             | status   |   count |
-|:-------------------------------------|:---------|--------:|
-| aten._log_softmax.default            | ✘        |       1 |
-| aten.addmm.default                   | ✅       |       2 |
-| aten.clone.default                   | ✅       |       2 |
-| aten.convolution.default             | ✘        |       2 |
-| aten.max_pool2d_with_indices.default | ✘        |       1 |
-| aten.relu.default                    | ✅       |       3 |
-| aten.t.default                       | ✅       |       2 |
-| aten.view.default                    | ✅       |       1 |
```
Comment (ayerofieiev-tt) [inline README.md:50]: If we remove these, we lose valuable info from reports, right?
Lets not remove till we gain that in cumulative and model pages

File: tests/conftest.py [hunk 2]
```diff
@@ -70,14 +71,30 @@ def compile_and_run(device, reset_torch_dynamo, request):
...
+            # Rerun with bypass option to collect aten op metrics
+            torch._dynamo.reset()
+            option.bypass = True
+            m = torch.compile(model, backend=torch_ttnn.backend, options=option)
+            if isinstance(inputs, collections.Mapping):
+                outputs = m(**inputs)
+            elif isinstance(inputs, collections.Sequence):
+                outputs = m(*inputs)
+            else:
+                outputs = m(inputs)
```
Comment (ayerofieiev-tt) [inline tests/conftest.py:87]: move to another method

File: tools/collect_metrics.py [hunk 3]
```diff
@@ -44,6 +45,32 @@
 }
 
 
+class ConversionStatus(Enum):
+    DONE = (1,)
+    FALLBACK = (2,)
```
Comment (ayerofieiev-tt) [inline tools/collect_metrics.py:50]: We might need another option - NONE.
FALLBACK is valid is there is some conversion (means at least 1 input variation for this op has status DONE), but this particular input is not covered and falls back to torch.

File: README.md [hunk 4]
```diff
@@ -8,17 +8,27 @@ This project allows to run PyTorch code on [Tenstorrent](https://tenstorrent.com
...
+| Model                                                                            | Run Success   | Torch Ops Before (Unique Ops)   | Torch Ops Remain (Unique Ops)   | To/From Device Ops   |   Original Run Time (ms) | Compiled Run Time (ms)   | Accuracy (%)   |
+|:---------------------------------------------------------------------------------|:--------------|:--------------------------------|:--------------------------------|:---------------------|-------------------------:|:-------------------------|:---------------|
+| [Mnist (Eval)](docs/models/Mnist (Eval))                                         | ✅            | 14 (8)                          | 5 (4)                           | 16                   |                    34.84 | 494.78                   | 99.7           |
+| [facebook/deit-base-patch16-224](docs/models/facebook/deit-base-patch16-224)     | ✅            | 685 (17)                        | 258 (6)                         | 723                  |                  1206.61 | 22429.53                 | 96.01          |
```
Comment (ayerofieiev-tt) [inline README.md:14]: I wish this did not separate mnist models. Maybe sort by name

File: torch_ttnn/backend.py [hunk 5]
```diff
@@ -123,8 +130,7 @@ def aten_backend(
     gm.recompile()
 
     if option.metrics_path:
-        compiled_schema_list = metrics.collect_schema_from_nodes(gm.graph.nodes)
-        metrics.save_pickle(compiled_schema_list, option.metrics_path, "compiled-schema_list")
+        option.compiled_schema_list.extend(metrics.collect_schema_from_nodes(gm.graph.nodes))
```
Comment (ayerofieiev-tt) [inline torch_ttnn/backend.py:133]: compiled schema contains schema for TT-NN ops?

**Review by ayerofieiev-tt — APPROVED** [review]

File: torch_ttnn/backend.py (hunk 5 above)
Comment (kevinwuTT) [reply torch_ttnn/backend.py:133]: Yeah this part holds the graphs containing ttnn ops

File: README.md (hunk 1 above)
Comment (kevinwuTT) [reply README.md]: Added these tables back. Only models that compile and run will show up.

File: README.md (hunk 4 above)
Comment (kevinwuTT) [reply README.md]: Table is sorted alphabetically now.

**Pull request merged**